tests/support_server/setup.pm

```diff
@@ -490,6 +491,29 @@ sub setup_mariadb_server {
     $disable_firewall = 1;
 }
 
+sub setup_nfs_server {
+    my $nfs_mount       = "/nfs/shared";
+    my $nfs_permissions = "rw,sync,no_root_squash";
+
+    # Added as the client test code might want to change the default
+    # values
+    if (get_var("CONFIGURE_NFS_SERVER")) {
+        $nfs_mount       = get_required_var("NFS_MOUNT");
+        $nfs_permissions = get_required_var("NFS_PERMISSIONS");
+    }
+
+    zypper_call('in rpcbind nfs-kernel-server');
+    systemctl("start rpcbind");
+    systemctl("start nfs-server");
+    assert_script_run("nfsstat –s");
+    assert_script_run("mkdir -p $nfs_mount");
+    assert_script_run("chmod 777 $nfs_mount");
+    assert_script_run("echo $nfs_mount 10.0.2.2/24\\($nfs_permissions\\) >> /etc/exports");
+    assert_script_run("exportfs -r");
+    systemctl("restart nfs-server");
+    systemctl("restart rpcbind");
+}
```

Comment:
Please add `systemctl status` check to see/check that services really started.

Reply:
@czerw 
will this do the job?
http://10.160.66.198/tests/4251#step/setup/64
http://10.160.66.198/tests/4251#step/setup/65